schema/admins/area.yaml

```diff
@@ -0,0 +1,26 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Area Schema

```

Comment:
I think it would help clarify that this is _Locality_ Area Schema at least here in the title. 

Not sure if it's worth considering or not because it's a minor thing, so i'll leave that up to you but if we care to be consistent with when we qualify the _kind_ of feature, then we should call this new type `localityArea`, similar to how we have `administrativeBoundary`, or conversely rename that to `boundary`.

Reply:
+1 to localityArea or administrativeArea

Reply:
Done.